tests/main/ubuntu-core-snapd-run-from-snap/task.yaml

```diff
@@ -20,7 +20,7 @@ restore: |
     umount /snap/snapd/current/usr/lib/snapd || true
     # and ensure snapd is working (needed in restore)
     systemctl daemon-reload
-    systemctl start snapd.service snapd.socket
+    systemctl start snapd.service snapd.socket snapd.snap-repair.timer
```

Comment:
Thanks for chasing this down! Looking at `snapd.run-from-snap` - won't we have to start a bunch more units? I see 
```
snapd.service snapd.autoimport.service snapd.core-fixup.service \
       snapd.seeded.service snapd.snap-repair.service \
       snapd.snap-repair.timer snapd.socket"
```
in there.